VisualizationBase/src/nodes/InfoNode.cpp

```diff
@@ -73,11 +73,19 @@ void InfoNode::updateInfo(bool isAutoUpdate)
 			cachedInfoStrings_[name] = getter.getter_(target_);
 		//Use the computed value to extend the info
 		if (!cachedInfoStrings_[name].isEmpty())
-			infoHtml += cachedInfoStrings_[name] + "<small>" + "  (Layer " + name + ")</small>" + "<br>";
+			infoHtml += cachedInfoStrings_[name] + "<small>" + "  (Layer " + name + ")</small>" + "<br><hr>";
 	}
+	infoHtml = "<html>" + infoHtml + "</div></html>";
 	setInfoHtml(infoHtml);
 }
 
+void InfoNode::setInfoHtml(QString content)
+{
+	if (content == infoHtml_) return;
+	infoHtml_ = content;
+	contentChanged_ = true;
```

Comment:
For this to work as expected, we'll probably have to tweak when VInfoNode is updated. I'll put a comment there.